### PR TITLE
PLG-317: add fake EE features in PIM menus and dashboards for the free trial

### PR DESCRIPTION
**Pull request by rouliane**
image
image
image

### CODE REVIEW

File: front-packages/shared/src/components/Navigation/PimNavigation.tsx [hunk 1]
```diff
@@ -2,9 +2,11 @@ import React, {FC, useMemo} from 'react';
 import styled from 'styled-components';
 import {PimView} from '../PimView';
 import {useRouter, useTranslate} from '../../hooks';
-import {IconProps, MainNavigationItem} from 'akeneo-design-system';
+import {IconProps, LockIcon, MainNavigationItem, Tag} from 'akeneo-design-system';
 import {SubNavigation, SubNavigationEntry, SubNavigationType} from './SubNavigation';
 
+const FeatureFlags = require('pim/feature-flags');
```
Comment (juliensnz) [inline front-packages/shared/src/components/Navigation/PimNavigation.tsx:8]: 😬 using require is not possible in the shared component :)

Comment (rouliane) [reply front-packages/shared/src/components/Navigation/PimNavigation.tsx:8]: Indeed, thanks, I'll do it differently.

**Review by LaurentPetard — APPROVED** [review]

File: src/Akeneo/FreeTrial/back/Infrastructure/Symfony/Resources/config/form_extensions/menu.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    config:
+      title: free_trial.menu.assets
+      icon: AssetsIcon
+      disabled: true
+
+  #activity sub menu > proposals
+  pim-free-trial-activity-submenu-proposals:
+    module: pim/menu/item
+    parent: pim-menu-activity-navigation-block
+    position: 104
```
Comment (olivierpontier) [inline src/Akeneo/FreeTrial/back/Infrastructure/Symfony/Resources/config/form_extensions/menu.yml:30]:
```suggestion
    position: 104
    feature: free_trial
```

File: src/Akeneo/FreeTrial/back/Infrastructure/Symfony/Resources/config/form_extensions/menu.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    config:
+      title: free_trial.menu.proposals
+      disabled: true
+
+  #System sub menu > SSO
+  pim-free-trial-system-submenu-sso:
+    module: pim/menu/item
+    parent: pim-menu-system-navigation-block
+    position: 125
```
Comment (olivierpontier) [inline src/Akeneo/FreeTrial/back/Infrastructure/Symfony/Resources/config/form_extensions/menu.yml:39]:
```suggestion
    position: 125
    feature: free_trial
```

File: src/Akeneo/FreeTrial/back/Infrastructure/Symfony/Resources/config/form_extensions/menu.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    config:
+      title: free_trial.menu.sso
+      disabled: true
+
+  #System sub menu > user groups
+  pim-menu-system-user-group:
+    module: pim/menu/item
+    parent: pim-menu-user-navigation-block
+    position: 140
```
Comment (olivierpontier) [inline src/Akeneo/FreeTrial/back/Infrastructure/Symfony/Resources/config/form_extensions/menu.yml:48]:
```suggestion
    position: 140
    feature: free_trial
```

File: src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/components/SettingsIndex.tsx [hunk 5]
```diff
@@ -268,6 +271,13 @@ const SettingsIndex = () => {
                     }
                   />
                 )}
+                {featureFlags.isEnabled('free_trial') &&
+                  <DisableIconCard
+                    icon={<LockIconContainer><LockIcon size={16} color={'#5992c7'}/></LockIconContainer>}
```
Comment (olivierpontier) [inline src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/components/SettingsIndex.tsx:276]: Could we defined colors used in the FT in a dedicated theme file to share it with the components that need these colors?

**Review by olivierpontier — APPROVED** [review]
GTM after fix

**Comment by olivierpontier** [comment]
We have several failing static tests when we run the CI: https://app.circleci.com/pipelines/github/akeneo/pim-enterprise-dev/41746/workflows/8b209127-19a7-4d96-98ae-a2f0807afce1/jobs/163946/parallel-runs/0/steps/0-104
@rouliane could you take a look on it please?

File: src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/components/SettingsIndex.tsx (hunk 5 above)
Comment (olivierpontier) [reply src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/components/SettingsIndex.tsx]: We will re-think about using a dedicated "theme" constantes for the Free Trial later in a dedicated PR

**Review by BryanS29-akeneo — APPROVED** [review]

**Comment by rouliane** [comment]
warmup

**Pull request merged**